docs/generator.go

```diff
@@ -0,0 +1,685 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"text/template"
+)
+
+func main() {
+	contents, err := ioutil.ReadFile("../schema.json")
```

Comment:
This is a detail, but maybe you could already parse a command line argument to get the file's path